dingo/backup/redundant_facet_removal_backup.py

```diff
@@ -0,0 +1,392 @@
+
+                            if width < redundant_facet_tol:
+                                offset += 1
+                                Aeq_res = np.vstack(
+                                    (
+                                        Aeq_res,
+                                        A[
+                                            i,
+                                        ],
+                                    )
+                                )
+                                beq_res = np.append(
+                                    beq_res, min(max_objective, min_objective)
+                                )
+                                ub[i] = sys.float_info.max
+                                lb[i] = -sys.float_info.max
+                            else:
+                                indices_iter.append(i)
+
+                                if not redundant_facet_left:
+                                    A_res = np.append(
+                                        A_res,
+                                        np.array(
+                                            [
+                                                A[
+                                                    n + i,
+                                                ]
+                                            ]
+                                        ),
+                                        axis=0,
+                                    )
+                                    b_res.append(b[n + i])
+                                else:
+                                    lb[i] = -sys.float_info.max
+
+                                if not redundant_facet_right:
+                                    A_res = np.append(
+                                        A_res,
+                                        np.array(
+                                            [
+                                                A[
+                                                    i,
+                                                ]
+                                            ]
+                                        ),
+                                        axis=0,
+                                    )
+                                    b_res.append(b[i])
+                                else:
+                                    ub[i] = sys.float_info.max
+                        else:
+                            ub[i] = sys.float_info.max
+                            lb[i] = -sys.float_info.max
+
+                b_res = np.asarray(b_res)
+                A_res = np.asarray(A_res, dtype="float")
+                A_res = np.ascontiguousarray(A_res, dtype="float")
+
+                return A_res, b_res, Aeq_res, beq_res
+
+    except gp.GurobiError as e:
+        print("Error code " + str(e.errno) + ": " + str(e))
+    except AttributeError:
+        print("Gurobi solver failed.")
+
+
+
+def fast_inner_ball(A, b):
```

Comment:
Could you plz also remove the function fast_inner_ball() from this file?

Reply:
okay

Reply:
Thanks!